### PR TITLE
Fixes #1080: Show 'No unread messages' if unread list is empty

### LINKED ISSUE #1080: "Unread" view should say something if there's nothing in it for a reason
When I have no unread messages, the "unread" view appears blank, which is pretty confusing and makes one think that's it's still loading something.

### PR DESCRIPTION
**Pull request by kunall17**
simulator screen shot 18-sep-2017 3 32 07 pm

### CODE REVIEW

File: src/i18n/translations/messages_en.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "No messages with this topic": "No messages with this topic",
   "No messages with this person": "No messages with this person",
   "No messages in this group": "No messages in this group",
+  "No unread Messages": "No unread messages",
```
Comment (borisyankov) [inline src/i18n/translations/messages_en.json]: Case is incorrect 'M' => 'm'

File: src/unread/UnreadStreamsCard.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+import { Label } from '../common';
 import { streamNarrow, topicNarrow } from '../utils/narrow';
 import StreamItem from '../streams/StreamItem';
 import TopicItem from '../streams/TopicItem';
 
+const componentStyles = StyleSheet.create({
+  emptyUnread: {
+    textAlign: 'center',
+    fontSize: 16,
+    marginTop: 5,
```
Comment (borisyankov) [inline src/unread/UnreadStreamsCard.js]: Add vertical centering instead of marginTop

File: src/unread/UnreadStreamsCard.js [hunk 3]
```diff
@@ -22,7 +31,9 @@ export default class UnreadStreamsCard extends PureComponent {
   render() {
     const { styles } = this.context;
     const { unreadStreamsAndTopics } = this.props;
-
+    if (!unreadStreamsAndTopics.length) {
```
Comment (borisyankov) [inline src/unread/UnreadStreamsCard.js]: `length === 0` is more clear

**Comment by kunall17** [comment]
Done

**Comment by borisyankov** [comment]
Cool.

**Pull request merged**